project1/src/components/Comments.tsx

```diff
@@ -18,34 +25,55 @@ interface commentsProps {
   comments: commentItem[];
 }
 
+const useStyles = makeStyles({
+  card: {
+    maxWidth: 600
+  }
+});
+
 const Comments: React.FC<commentsProps> = props => {
+  const classes = useStyles();
   const content = props.comments.map(item => {
     return (
       <div key={item.id} className="card">
-        <div className="card-block">
-          <p className="card-text">{item.body}</p>
-        </div>
-        <div className="card-footer">
-          <a className="comment-author" href="#/@yiwei">
-            <img src={item.author.image} className="comment-author-img" />
-          </a>
-          <a
-            className="comment-author ng-binding"
-            ui-sref="app.profile.main({ username: $ctrl.data.author.username })"
-            href="#/@yiwei"
-          >
-            {item.author.username}
-          </a>
-          <span className="date">
-            {" "}
-            {moment(new Date(Date.parse(item.createdAt))).format(
-              "MMMM D, YYYY"
-            )}
-          </span>
-          {/* <span className="mod-options ng-hide" ng-show="$ctrl.canModify">
-            <i className="ion-trash-a" ng-click="$ctrl.deleteCb()"></i>
-          </span> */}
-        </div>
+        <Card className={classes.card}>
+          <CardContent>
+            <Typography gutterBottom variant="h5" component="h2">
+              <a className="comment-author" href="#/@yiwei">
```

Comment:
what does the href here mean?

Reply:
I will edit it later, this link should go to the article's author profile.